dicom-mr-classifier.py

```diff
@@ -579,7 +579,7 @@ def dicom_classify(zip_file_path, outbase, timezone, config=None):
         metadata["acquisition"]["label"] = series_desc
         classification = get_custom_classification(series_desc, config)
         log.info("Custom classification from config: %s", classification)
-        if not classification:
+        if (not classification) and ('MR' in dcm.get("Modality")):
```

Comment:
Nit: 
```suggestion
        if not classification and dcm.get("Modality") == "MR":
```

Just in case we ever bump into some weird DICOM with custom Modality defined having "MR" in it. (that does not seem to be the case for the modality in the standard http://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_29.html)